src/Config.js

```diff
@@ -71,13 +72,15 @@ export default class Config {
   fileContents: string;
   json: JSONValue;
   indent: string;
+  newline: string;
   invalid: boolean;
 
   constructor(filePath: string, fileContents: string) {
     this.filePath = filePath;
     this.fileContents = fileContents;
     try {
       this.indent = detectIndent(fileContents).indent || '  ';
+      this.newline = detectNewline(fileContents) || '\n';
```

Comment:
Should we use `os.EOL`? ( It is a question because I am sure if that has any advantage over having `'\n'` )

Reply:
`os.EOL` would be a dynamic value. I think the idea here is to keep the line-ending style in `package.json` consistent even if folks working on different OSes are making changes to this file.